nlp_apps.ipynb

```diff
@@ -559,6 +586,63 @@
knxkmXHOJA0VrqMnqoQK/qE7oRhelMElf1QpZe3yB0N+MZKyVJ5Uy6jfBnrhZArAJ6bpVYRRgDhl+5eJDZqadZqbw+6KxunidDWFjSIdIBjpEjZebw/R4pI0/ayKN1TKQDLJnEDlrv7CT0IBAy0WawTMHcrCSONIYJU5MGnqcxJHS7Cqu+liTWKkZIA7s4Nky0++sLqNBHRxG95WvogThgf30JFfpJLKXu7RIDScf5djZBQu/eZXqhkxCuWLw7ezPWEV29w1jpLpdqeBtLLOUUyWUmN+nUKqLbW7uvivy2YxqMKtxKzJPnLmBbOYLw3p33ldWYz33BRRrQaGq00ng+VCoevb5ieTUinZJUqzG9vuYrX/eQMilgrG8ogkBz42ZEEMLCUkzga2IFn/+iS3sHh413i6DXZP3cdyhMH8Gw06yc+RpBv5kIRgK5yYMUpg+zevYbeJ0qubF9TBx/nu7GDfzWBuWDj6OigOU3voSKAqZPfxwzlXwHrHSeiWPP0V2/TuPmWQwny8yjn6S07xEqF14EQFopnHSelTe/gt+tIw2TYNBgC/stVt/6OnGYNGemT3+ckbmTd03M1WAMI96lrR0kxIKRMfNHrin8Y5WY52dyjBwY4frXb6AihYoVYUciN9aTTdTaXQXuQRD2QupXmjSuNbnypXnGT4wy98wMk6fHmH5ikrnn9tBZ6XLzu0ssf3+NjfO1Lb6iQ2gIKh5RLcQSDqNiBld1sJYTO4iCyKClQlYMcpQJhYeoSkwsqIGqxfTjLuFAOKK/2qG/uvXL7dX6eLWtAaXfcPEbLhlzhKw5gi3TyJaBI3NoW+EYGaQ2cBe7RHFA0R4jZ4wTOC6u16F/rUs3Si54B/bB+IzPjUs+4zMm8WQKyxEURkJUDPKIzeW3YnqdHjOHHcJAY9kxcUtS6cf03Q1anYVEBG5ApArDHhrodJcHAXtC4up0Vwan7W78152ho5D4Loa26i6BaXfx0uAlktdJWwX2FR+n7VdYap+9q53cnXAO7EHmMqRPHhwoH0uMbI6edwlSW/3PhSGRpkn97Vcwszma536At7FCbt9ROleTZEiY5nuuGg+JSKE5Ooo9N4d982Yyb75vH/7SEkpDadQgkx3YuVgm5vgIYaUBa0miZJgOPb1E0G8hZKL+Lw2bKHAHSpkKt1NFqwgVBYnY022MgajfoTN/gYmnPs7E+z/JhjTor97YeozSwClPUn7kWbKzh/AbG/QWtl5QnXSRjcU3KYzuR2gDvWtF+59hJ4SrVRqf/mYyosKAuXNrJlhKUuVpojMrdC+3oO9hF8YwnHTyGYcB4TsrxJcqCCOhRMauS7o8g45j3JcvYmbypEcmQUqCdn0bA+KWi0Uwv5LYrkmR+IOv1zHLSfFP2CbhWj0pEAlBYe4E2fG9STICmE6G0qHHqF5+lbieBFIy7WCWR3BfP4vMZZEpm3BtA2tqAiEEcbOD6ruE61XiVhvLsTEKeaLq7tSKdwUNfrWLX+tS/f5N1l+8Rn+xseX+2zFl7CMlshhUSInM5oz5jxgnH3Fwe5qHTjlcOBvw5HMprl+LmDmao9cIiUJFc92nNJ0CDflRi/qKx5H3jVBb9lif7zM6myL0E/2LVNYg9BXr1/vE0W2JS3mEuH5HYC0E1twM2aakGJboxU0EAjfYnphbFjx82uL0IyAF3LwZUyonM5mFQkIX/863fa5cuTV2FeEu3UBYFlGniTAthGGBVmiV/EmYGho16OToSFE/uzKUIBhKEWhNb7HJtjsE6FhTP7M87HgHnuL6Oy5eLyaVNTAsQacZoWJNv6MY5NQIoNuKUJEmCrYujsxomjiIUXGU0OmByOthOhnCfgczlUVISRx4hF4vCUwHB+W1AlJFJymaKE1n3aVf9bCzFm7DRymN2/C3BZqlmTRHnx3ljS+s3lNh+acJ1cZlpsYfIZMqA0kXeP+eD1DMz7K2cZZGex5/sFbduI0Xd4Zfc+EnsU03qt9mpAudsDr4+YeP9D1P8+0Xkz1RysG6SmroQx9npSBcX9hSKI36XXaKuXbSAdot1ECd/nbc0nO63V86UzDp1kOkyXDdC5GwePL5H413u23nSadGhv+PVYTr17cd//2gtSIMN+NxIQSGtHHs+yfmYQg3drCc9LzkGNbWFNwWl67cTk0fPObWK1+7vhlDVjbeY6tDrVBRQBwFg599VLjZ/MxNHsB0sqTLM6RL0xiWg5nOkR6ZIug1yZRnaS6+g9dK2Avt1auM5UeBhKaeLs/gdaqM7H04sdM1LTKje5CGmVz/hKBXXaRXW9o22naLmp8b35+IWqazu/sIDTmIJ5KCCCKJR4VtDTVyhEjGPN2eYuGKz6PPZrl4xiUaKLa7fUWv/VdnK/ljlZirSBH2Q8J+SOwnc9TK1fjLy6iSm1x9doA0LNKFKfLlvZip3NBXcSesX3+ZoL+98qYVhP2IldfXqV6qM358lCO/cIB9H5yluDfPw79xjCO/cICNC3WufGmexZdXCHtbN7CYiHac0H0lSSApuUXl1cNZo1se6hKJQKKIkRiE+t0nHn7cJVYhUkg0EOsweX4hEyrw4EseeAm9SqMT4YXbzlOrHrO2GFBdjVhblKAhk5eMTlqs3gyIoiS4MAzoNGNMS1CeMHF7yXN7wXZBBT1892zZHG7dbggLQ1oIBLGOiFSAIUwMaSX0ZeUn51OYyXvTemAvpDGlgxQGSsfJ45CJMMhgtj2IXaSQmDLxb4xUgNIR3aBKtX8D20jzIHDfuYo9N4V/fZmo2kDYFtknTuB2KrjdrV0/rRSR28NwUsRun5GT70Oam183c7SEUS4T3FjY0b5vCEMiTAPt7z5YEoaBPTWFVS6TeThRQteuS7C+jtCQSgv6g8/MKOUpfuxx6p/5LmpQMY4jn159BxG/XULHMc3zr5GenCN/4DgzH/81gsYGdnEUYVqUH36a4pGHsUfGsUdGidwe1de/TdDe2gn3ejXyo/uIQneoyC5SDsKyEsZEkI
```

Comment:
Replace the '&' with 'and' and remove the 'the' from 'Hamilton seems to use the words'. Also, when you are referencing a word put it in quotes. Like this:

They all have common words like "state", "will" and "government".

Finally, the final sentence ('Word clouds reveal some sort of sentiments if analyzed properly.') is a bit too vague. You need to add some more details, right now it doesn't carry much information.